### PR TITLE
add task stop watch solution

### PR DESCRIPTION
**Pull request by opiquem**
[DEMO LINK](https://maksym-bobryk.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/styles/utils/mixins.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  animation: rotate-hand $time infinite linear;
+}
+
+@mixin dot ($w, $h, $color, $radius) {
+  width: $w;
+  height: $h;
+  
+  background-color: $color;
+  border-radius: $radius;
+}
```
Comment (cephy77) [inline src/styles/utils/mixins.scss:20]: In my opinion, You can just add these styles to clock.scss

**Review by cephy77 — APPROVED** [review]
Good one!

File: src/styles/blocks/clock.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.clock {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 500px;
+  height: 500px;
+  border: 2px solid #000;
+  border-radius: 50%;
```
Comment (Ihor-Karpyn) [inline src/styles/blocks/clock.scss:9]: Mixin circle will be useful here

File: src/styles/utils/mixins.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
+@mixin hand ($w, $h, $color, $time) {
+  position: absolute;
+  bottom: 50%;
+
+  width: $w;
+  height: $h;
```
Comment (Ihor-Karpyn) [inline src/styles/utils/mixins.scss:6]: Mixin square

File: src/styles/utils/mixins.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+  background: $color;
+
+  transform-origin: bottom;
+  animation: rotate-hand $time infinite linear;
+}
+
+@mixin dot ($w, $h, $color, $radius) {
+  width: $w;
+  height: $h;
```
Comment (Ihor-Karpyn) [inline src/styles/utils/mixins.scss:16]: square again

**Review by Ihor-Karpyn — APPROVED** [review]
GJ

**Pull request closed without merging**